cmd/ebctl/sboms.go

```diff
@@ -307,6 +307,7 @@ type UploadSBOMForCIArgs struct {
 
 type UploadSBOMForCIOutput struct {
 	CommentBody string `json:"comment_body"`
+	SkipComment bool
```

Comment:
This needs a struct tag, similar to the line above. Should look something like:

```go
	SkipComment bool `json:"skip_comment"`
```

Struct tags are weird, but the tl;dr is that when we serialize this struct to JSON the serializer will look at the struct tag and use it to give a name to this field. It's mostly a stylistic thing, but without it the JSON would be like:

```json
{
    "comment_body": ".... whatever ....",
    "SkipComment": true
}